tests/kuttl/test-cyndi-strimzi/01-install.yaml

```diff
@@ -43,6 +43,8 @@ metadata:
 spec:
   cyndi:
     enabled: true
+    additionalFilters: 
```

Comment:
Did you check that this doesn't break if additionalFilters is missing?

Reply:
We have a default value set to `[]˙ in our app, if that helps

Reply:
doesn't break if it's empty or missing either (tested both on my local minikube)